scripts/release_helper/release_helper.yml

```diff
@@ -54,10 +54,8 @@ jobs:
 
         # import env variable
         export TOKEN=$(Yuchao-GitToken)
-        export PYTHON_BIGCAT_TOKEN=$(Jiefeng-GitToken)
-        export PYTHON_ZED_TOKEN=$(Zed-GitToken)
-        export PYTHON_MSYYC_TOKEN=$(Yuchao-GitToken)
         export AZURESDK_BOT_TOKEN=$(azuresdk-github-pat)
+        export PIPELINE_TOKEN=$(PIPELINE-TOKEN)
```

Comment:
`AZURE_BOT_TOKEN` does not have access to trigger pipeline?

Reply:
yes, the Azure_BOT_TOKEN is github token, but it needs Azure DevOps token.

Reply:
> `AZURE_BOT_TOKEN` does not have access to trigger pipeline?

yes, the Azure_BOT_TOKEN is github token, but it needs Azure DevOps token.